gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,6 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.6-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.4-bin.zip
+networkTimeout=10000
```

Comment:
Did you explicitly add this timeout? If so, why set it to 10s?

Reply:
This was automatically added by the gradle update script